frontend/src/views/DiagnosticTunnel/index.vue

```diff
@@ -360,6 +361,10 @@ export default {
     quit() {
       this.$router.push(this.quitLink)
     },
+    onFullTunnelAutofill(e) {
+      this.updatePayload({ payload: e.payload, formIsValid: true })
+      this.saveAndQuit()
```

Comment:
IMO we should go to the synthesis step in the tunnel rather than quit completely. I think this will be less jarring, and implicitly explain that the button helped complete the steps, rather than it being some strange bug and that you need to go back and complete the tunnel. Also it would mean that we don't have to worry about changing the tunnel completion status in the backend with this PR.

Reply:
Fix here : https://github.com/betagouv/ma-cantine/pull/3469/commits/04fea22a11de04ecbfbc7cb87cfbecd36f5aa903